src/language-handlebars/utils.js

```diff
@@ -25,7 +25,7 @@ function isLastNodeOfSiblings(path) {
 }
 
 function isUppercase(string) {
-  return string.toUpperCase() === string;
+  return /^[A-Z]+$/.test(string);
```

Comment:
If it contains a number, how should we print it? Is there a test for it? Or maybe contains `-`.

Reply:
This is how 2.5 prints it: https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAjAMwhABAegD4AdKVAWmwiNNSprIDEciQAaECABxgEtoAzslABDAE5iIAdwAK4hEJQiANlJEBPIRwBGYkWADWcGAGURAWzgAZXlDjJMKgXB17Dxk1322A5shhiAK4uIHDm2nAAJpFRViJQPoEiPnDMYuYiMHwJyCAigTAQ7CAAFjDmygDqJbzwAl5gcCYKtbwAbrXquWACWiC2zmIwMno+GQ5OIQBWAgAeJr7KcACKgRDwE8rOHF5ig7k+yrzmlmLFXGK2MJW8kTAlyAAcAAw7ks6Vely5F3CDbfYOABHNbwEbcRR5ATkOxRKLFMRwEG8REjZLjJCOLYhZzmXj+II4xYrUH2TGTDgwETaG53B5IABMlL0vCOCQAwhATiJcn8AKzFQLOAAq1MUWO2IDawQAklAYrATGBLjwAILykwwdRLTbOAC+eqAA

Reply:
For clarification: this is checking whether the tag is a Glimmer component or not. So to be strict, a named block `:foo` isn't a Glimmer component so we shouldn't print as one

Things starting with a number or `-` aren't even valid tags so I think it's out of the scope of this PR

Reply:
Ok, got it now. I'll change the implementation

Reply:
@fisker let me know if it looks better now